src/SbManager/Scripts/manager/homeController.js

```diff
@@ -1,9 +1,17 @@
 ﻿$app.controller('homeController', ['$scope', '$routeParams', function ($scope, $routeParams) {
     $scope.refresh = function () {
         $scope.model = null;
-        $.getJSON(window.applicationBasePath + "/api/v1/busmanager/", {}, function (d) {
-            $scope.model = d;
-            $scope.$digest();
+        $.ajax({
```

Comment:
We should be using $http not jQuery here. I know the previous version violated this but if you are going to change it I think we should make it right. What do you think?